src/twisted/internet/defer.py

```diff
@@ -2226,7 +2227,7 @@ def unwindGenerator(*args: _P.args, **kwargs: _P.kwargs) -> Deferred[_T]:
 
 
 _ConcurrencyPrimitiveT = TypeVar(
-    "_ConcurrencyPrimitiveT", bound="_ConcurrencyPrimitive"
+    "_ConcurrencyPrimitiveT", bound="_ConcurrencyPrimitive[Any]"
```

Comment:
Why is _ConcurrencyPrimitive even generic? ` _SelfResultT` is only used in `def run` where it would be valid without a class level Generic

Reply:
Just go with whatever you think is best. Thanks :)

Reply:
@adiroiban I have a PR removing the Generic and fixing up the type here https://github.com/twisted/twisted/pull/11986